web/components/post-card.tsx

```diff
@@ -45,6 +46,15 @@ export function PostCard(props: {
             <span className="text-gray-500">{fromNow(post.createdTime)}</span>
           </div>
           <div className="text-lg font-medium text-gray-900">{post.title}</div>
+          <div className="font-small text-md text-gray-500">
```

Comment:
I like these changes for the post card! A subtitle plus a little post tag + document icon is nice!

Reply:
I would add more padding and especially left padding to the card – currently seems unbalanced.

E.g. https://twitter.com/danqing_liu/status/1576997498434252800

Another idea is to set a max width, e.g. `max-w-[300px]`, so that it doesn't look weird extending to the right.